Makefile

```diff
@@ -4,16 +4,20 @@ DEVTAG=$(DRIVER_VERSION)-dev
 NAME=oras-csi-plugin
 DOCKER_REGISTRY=ghcr.io/converged-computing
 
-ready: clean compile
-publish: uninstall compile build push install
-dev: uninstall compile build-dev push-dev install
+.PHONY: help
+help: ## Generates help for all targets
+	@grep -E '^[^#[:space:]].*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
I love this! Looks great - I should add help commands like this to all my Makefile.

![image](https://user-images.githubusercontent.com/814322/231597818-47a0050f-53be-44d4-8150-4634df3a85fb.png)